base/fitness_model.py

```diff
@@ -377,15 +414,25 @@ def clade_fit(self, params):
             clade_errors = []
```

Comment:
Why aren't we calculating `self.projection(params, self.predictor_arrays[time], self.freq_arrays[time], self.delta_time)` once and slice the array later when you do it clade by clade?